.github/dependabot.yaml

```
@@ -146,6 +146,10 @@ updates:
       # requires to also bump `TEST_INFRA_DEFINITIONS_BUILDIMAGES` inside `.gitlab/common/test_infra_version.yml`
       # and dependabot isn’t able to keep those two synchronized.
       - dependency-name: github.com/DataDog/test-infra-definitions
+      # Do not bump dependencies that come from test-infra-definitions, they will be bumped when we'll bump the version
+      # https://datadoghq.atlassian.net/browse/ADXT-375
+      - dependency-name: github.com/pulumi/*
```

To include [pulumiverse/pulumi-time](https://github.com/DataDog/datadog-agent/pull/27440/files#diff-fee3041315eef2d6446d1e0f565e42851503923fb40e53cf4c91ac5641ebc4e0R185) 
```suggestion
      - dependency-name: github.com/pulumi*
```